boards/arm/qemu_cortex_m3/doc/board.rst

```diff
@@ -6,9 +6,8 @@ ARM Cortex-M3 Emulation (QEMU)
 Overview
 ********
 
-The Zephyr kernel uses the qemu_cortex_m3 board configuration to emulate the TI
-LM3S6965 platform running on QEMU. It provides support for an ARM Cortex-M3 CPU
-and the following devices:
+This board configuration to emulate the TI LM3S6965 platform running on QEMU. It
```

Comment:
We don't know that "this board configuration" is, so I'd recommend:

      The `qemu_cortex_m3` board configuration target will use QEMU to emulate the TI LM3S6965
      platform.  This configuration provides support for an ARM Cortex-M3 CPU and these devices: